.github/helm-docs.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPTPATH=$(dirname "$0")
+HELM_DOCS_VERSION="0.15.0"
```

Comment:
Should we not set the version to a maintained version, latest stable is 18

Reply:
this is the version of the helm docs tool